README.md

```
@@ -11,196 +11,92 @@
-### Editor Setup — VSCode IntelliSense
+---
 
-When you run `make up`, `make install`, or `make add-*`, the `Makefile` automatically syncs the container's installed `node_modules` back to the host filesystem. This gives VS Code running on your host machine instant access to all TypeScript type definitions and Biome formatting without needing Node.js or Dev Containers installed on your host.
+## API Documentation
 
-If you ever need to manually re-sync module definitions:
+NIddle includes built-in interactive Swagger documentation. When the backend API is running, access the documentation at:
 
-```bash
-make sync-modules
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)